tests/test_core/test_annotationdb.py

```diff
@@ -0,0 +1,633 @@
+from cogent3 import load_unaligned_seqs, open_
+from cogent3.core.annotation_db import GenbankAnnotationDb, GffAnnotationDb
+from cogent3.parse.genbank import MinimalGenbankParser
+from cogent3.parse.gff import gff_parser
+import os
+
+base_path = os.path.dirname(os.path.dirname(__file__))
+data_path = os.path.join(base_path, "data")
+gff_path = os.path.join(data_path, "prok_NoLocusTags.gff")
+genbank_path = os.path.join(data_path, "NC_000913.3.gb")
```

Comment:
The existing `tests/data/annotated_seq.gb` can be used for these tests. This means you don't need to add a new data file, which helps keep the repository more compact.